doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+.. _ug_thingy91:
+
+.. _thingy91_ug_intro:
+
+Working with Thingy:91
+######################
+
+Thingy:91 is a battery-operated prototyping platform for cellular IoT systems, designed especially for asset tracking applications and environmental monitoring.
```

Comment:
```suggestion
Nordic Thingy:91 is a battery-operated prototyping platform for cellular IoT systems, designed especially for asset tracking applications and environmental monitoring.
```
Let's use the full name on the first occurrence.

Reply:
Added.